library/borgbase.py

```diff
@@ -82,6 +82,42 @@ def add_repo(key_id):
     res = client.execute(REPO_ADD, new_repo_vars)
     return res
 
+def get_repo_id(name):
+    res = client.execute(REPO_DETAILS)
+    for repo in res['data']['repoList']:
+        if repo['name'] == name:
+            repo_id = repo['id']
+            return True, repo_id
```

Comment:
No need to return 2 values here: If the repo is found, return the name (truthy value). If not, the function will always return `None`, which can be checked below:

```
repo_id = get_repo_id(module.params['repository_name'])
if repo_id is None:
   # Add new repo using the key
   ...
else:
    # Edit the repo
   ...
```

Reply:
done